src/devices/weten.ts

```diff
@@ -11,6 +16,37 @@ const definitions: Definition[] = [
         extend: extend.switch(),
         fromZigbee: [fz.on_off, fz.ignore_basic_report, fz.ignore_time_read],
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_6fk3gewc']),
+        model: 'PCI E',
+        vendor: 'WETEN',
+        description: 'PC switch',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            e.binary('state', ea.STATE_SET, 'ON', 'OFF').withEndpoint('l1').withDescription('PC Power'),
+            e.binary('state', ea.STATE_SET, 'ON', 'OFF').withEndpoint('l2').withDescription('Shutdown or Reset? Does not seem to actually do anything'),
+            e.binary('state', ea.STATE_SET, 'ON', 'OFF').withEndpoint('l3').withDescription('Buzzer on means no buzzer noise'),
+		    e.binary('child_lock', ea.STATE_SET, 'LOCK', 'UNLOCK').withEndpoint('l4').withDescription('Child safety lock'),
+		    e.binary('state', ea.STATE_SET, 'ON', 'OFF').withEndpoint('l5').withDescription('To enable or disable the use of RF remote control, does not seem to actually work'),
+		    e.binary('state', ea.STATE_SET, 'ON', 'OFF').withEndpoint('l6').withDescription('To pair a RF 433 remote, such as the one supplied')
+        ],
+        endpoint: (device) => {
+            return {'l1': 1, 'l2': 1, 'l3': 1, 'l4': 1, 'l5': 1, 'l6': 1};
```

Comment:
Please remove this and don't use `state_lX`, instead do something like this:

```
e.binary('pair_rf_remote', ea.STATE_SET, 'ON', 'OFF').withEndpoint('l6').withDescription('To pair a RF 433 remote, such as the one supplied')
```

and

```
[103, 'pair_rf_remote', tuya.valueConverter.onOff],
```

Reply:
I tried that, however I get the following errors
`Received MQTT message on 'zigbee2mqtt/0xa4c138f82a910a04/set' with data '{"pair_rf_remote_l6":"ON"}'
Device '0xa4c138f82a910a04' has no endpoint 'l6''`

Reply:
Whoops, should be: `e.binary('pair_rf_remote', ea.STATE_SET, 'ON', 'OFF').withDescription('To pair a RF 433 remote, such as the one supplied')`

Reply:
Changing the code to be as such
```
    exposes: [
        e.binary('pc_power', ea.STATE_SET, 'ON', 'OFF').withDescription('PC Power'),		
        e.binary('shutdown_reset', ea.STATE_SET, 'ON', 'OFF').withDescription('Shutdown or Reset? Does not seem to actually do anything'),
        e.binary('buzzer', ea.STATE_SET, 'ON', 'OFF').withDescription('Buzzer on means no buzzer noise'),
	e.binary('child_lock', ea.STATE_SET, 'LOCK', 'UNLOCK').withDescription('Child safety lock'),
	e.binary('rf', ea.STATE_SET, 'ON', 'OFF').withDescription('To enable or disable the use of RF remote control, does not seem to actually work'),
	e.binary('pair_rf_remote', ea.STATE_SET, 'ON', 'OFF').withDescription('To pair a RF 433 remote, such as the one supplied')
        ],
     meta: {
        multiEndpoint: true,
        tuyaDatapoints: [
            [1, 'pc_power', tuya.valueConverter.onOff],
            [105, 'shutdown_reset', tuya.valueConverter.onOff],
            [104, 'buzzer', tuya.valueConverter.onOff],
            [106, 'child_lock', tuya.valueConverter.lockUnlock],
            [102, 'rf', tuya.valueConverter.onOff],
            [103, 'pair_rf_remote', tuya.valueConverter.onOff],
```

the only one that works is `child_lock`. All the others get an error such as `No converter available for 'pair_rf_remote' ("ON")`

Reply:
`multiEndpoint: true,` has to be removed.

Reply:
I did that but still got the error of `No converter available for 'pair_rf_remote' ("ON")`
But I might have found a solution. You can see my latest commit. I made a few changes and added
```
const tzDatapoints = {
    ...tuya.tz.datapoints,
        key: ['PC_Power', 'shutdown_reset', 'buzzer', 'child_lock', 'RF', 'pair_RF_remote']
}
```

and it's working now. Is that the right approach?

Reply:
please add these changes directly to the `datapoints` converter in `lib/tuya.ts` https://github.com/Koenkk/zigbee-herdsman-converters/blob/34de76fe3d1ffb2aa6dbcd712ceb5178f38712f9/src/lib/tuya.ts#L1097